schedule/yast/maintenance/create_hdd_transactional_server.yaml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
we cannot deploy directly in maintenance job group, I would prefer to see that everything is fine in development for 1 day, so I'm not sure why you remove the `_dev` postfix in the test suite name.

Reply:
I just debug in my own job group. To separate with yast maintenance  development job group cases' name, I add 'test_' to each case.

Reply:
If we can confirm schedule files' name, I will submit a new MR and change the cases' name.